### DISCUSSION TITLE
Welcome to ant-design-blazor Discussions!

### DISCUSSION THREAD

**Discussion by ElderJames** [ant-design-blazor/ant-design-blazor#872] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anddrzejb** [comment]
Let me start by saying I do enjoy this project immensely. It is clear it still needs a lot of work but I think it has a great potential if it manages to engage enough enthusiastic people. 
I have been programming for 14 years but only for around 1,5 years I've been doing it in C#. Blazor was the one that hooked me on board. I work for a very small company in Cyprus where I deal mostly with tourism and public transportation. 
What I would like to do is to transition the desktop apps we have to web app (maybe progressive as well) using blazor. And as I said earlier, I really like the way antDesign is presenting. Now that I actually tried to use it in my own project, I figured I will try to "kill 2 birds with 1 stone" - by contributing I will solve my issues and help the project grow. Plus the knowledge gained 😃 .

**Comment by ElderJames** [comment]
Thank you for liking AntDesign @anddrzejb . PR is always welcome! Community that makes the project good, this is the right direction to run the open source community.

**Comment by BoyJun** [comment]
麻烦维护一下组件库，https://ant-design-blazor.gitee.io/zh-CN/components/overview，我在打开Table和日历时会提示未绑定，导致页面崩溃，麻烦及时维护一下谢谢

**Reply by ElderJames** [reply]
It's OK now.
